src/constants/keyCodes.js

```diff
@@ -2,11 +2,12 @@ export const keyCodes = {
   ENTER: "Enter",
   SPACE: " ",
   ESCAPE: "Escape",
-  DOWN_ARROW: 40,
-  UP_ARROW: 38,
+  DOWN_ARROW: "ArrowDown",
+  UP_ARROW: "ArrowUp",
```

Comment:
just making sure no one else is using this so it won't break

Reply:
I've checked, yes